src/checker/index.ts

```diff
@@ -739,7 +745,6 @@ export class ImandraServerConn implements vscode.Disposable {
     const args = [
       ...(this.debug ? ["-d", "4"] : []),
       ...(this.config.persistentCache ? ["--persistent-cache"] : []),
-      ...(this.config.autoUpdate ? [] : ["--skip-update"]),
```

Comment:
should this be commented with a "todo" ? :)

Reply:
ok done